### PR TITLE
Add namespace

### PR DESCRIPTION
**Pull request by webwurst**
/ping @puja108 
Works for me:
```
kubectl apply --filename https://raw.githubusercontent.com/giantswarm/kubernetes-prometheus/add-namespace/manifests-all.yaml
```

### CODE REVIEW

**Comment by puja108** [comment]
LGTM, just fix the docs and we can merge

File: README.md [hunk 1]
```diff
@@ -4,45 +4,26 @@
...
-# minikube service prometheus
-
-kubectl create --filename manifests/prometheus-deployment.yaml
-# kubectl get --all-namespaces --output wide pods
-# kubectl logs prometheus-2556266794-sd260
-# kubectl delete pods/prometheus-2556266794-sd260
-
-kubectl create --filename manifests/node-exporter-service.yaml
-kubectl create --filename manifests/node-exporter-daemonset.yaml
+kubectl apply --filename ./manifests/grafana/grafana-import-dashboards-job.yaml
```
Comment (puja108) [inline README.md]: no need to delete the job?

**Comment by puja108** [comment]
1 last comment, otherwise go ahead

**Comment by webwurst** [comment]
@puja108 PTAL

**Pull request merged**